Code/Framework/AzCore/AzCore/std/string/string_view.h

```diff
@@ -11,12 +11,14 @@
 #include <AzCore/std/algorithm.h>
 #include <AzCore/std/createdestroy.h>
 #include <AzCore/std/iterator.h>
-#include <AzCore/std/hash.h>
+
 
 namespace AZStd
 {
     namespace StringInternal
     {
+        constexpr size_t minSize(size_t left, size_t right) { return (left < right) ? left : right; }
+
```

Comment:
This changes doesn't make much sense.

The `AZStd;:min` and `AZStd::max` functions comes from the algorithm.h header which is _still being included_

This file scope `minSize` function isn't needed

Reply:
This is in preparation for 'algorithm.h' removal from this header ( will remove it for now )

Reply:
Or better yet, I'll remove the algorithm.h right now and see what breaks 😄

Reply:
In that case, then the according to the [Coding Standard](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#function-names), the MinSize function should be UpperCamelCase.
Therefore can you name this `MinSize` instead

Reply:
Should this be min_size to match the file? rather than MinSize?

Reply:
Yeah `min_size` would work as well to match the surrounding style of the code.

Reply:
Done, algorithm.h removed, minSize renamed